src/components/LibraryItem.tsx

```diff
@@ -89,19 +89,27 @@ export class LibraryItem extends React.Component<LibraryItemProps, LibraryItemSt
             return null;
         }
 
-        let iconElement = null;
+        let iconElement: JSX.Element = null;
         let libraryItemTextStyle = "LibraryItemGroupText";
 
         // Group displays only text without icon.
-        if (this.props.data.itemType !== "group") {
+        if (this.props.data.itemType !== "group" && this.props.data.itemType !== "section") {
             libraryItemTextStyle = "LibraryItemText";
-            iconElement = (<img className={"LibraryItemIcon"} src={this.props.data.iconUrl} onError={this.onImageLoadFail} />);
+            iconElement = <img
+                className={"LibraryItemIcon"}
+                src={this.props.data.iconUrl}
+                onError={this.onImageLoadFail}
+            />;
         }
 
         // If it is a section, display the icon (if given) and provide a click interaction.
-        if (this.props.data.itemType === "section") {
-            iconElement = this.props.data.iconUrl ? (<img className={"LibraryItemIcon"} src={this.props.data.iconUrl} onError={this.onImageLoadFail}
-                onClick={this.onSectionIconClicked.bind(this)} />) : null;
+        if (this.props.data.itemType === "section" && this.props.data.iconUrl) {
+            iconElement = <img
+                className={"LibraryItemIcon"}
+                src={this.props.data.iconUrl}
+                onError={this.onImageLoadFail}
+                onClick={this.onSectionIconClicked.bind(this)}
```

Comment:
If you can move `onSectionIconClicked.bind` to where `onImageLoadFail.bind` was done that'll make this neater 😄